features/GEM005_Building-shape-representation.feature

```diff
@@ -6,8 +6,7 @@ The rule verifies that an IfcBuilding has a correct representation
 
   Scenario: Agreement on empty IfcBuilding using correct representation
 
-      Given A file with Schema Identifier "IFC2X3" or "IFC4" or "IFC4X3_TC1" or "IFC4X3_ADD1" or "IFC4X3"
-      And An IfcBuilding
+      Given An IfcBuilding
```

Comment:
While I'm of course fine with leaving this as you propose, just a thought. Do we care what happens when IFC 5.0 comes out?  The rule written without the Schema identifier will fire for every subsequent IFC version. For most (all?) rules this might be what we want, for some perhaps not.

Reply:
I'm not sure. On the one hand, IFC 5 might take a while to come. On the other hand, that also means that we might have long forgotten about this. 

In alternative implementations, if the `Given` clause begins with an IFC instance, such as `IfcBuilding`, it is understood that the clause applies to all IFC schemas.

Should we use our decorator for this purpose? We could check in the stack frame whether the first `Given` statement is one of the statements referring to versions. If not, use a default set of versions. In our case, that would be IFC Schema `IFC2X3`, `IFC4` and `IFC4X3`. 
When IFC5 comes, or a new IFC4 version, we'll be able to make modifications at this central 'handling' location instead of spreaded accross `.feature` files.